docs/3.1-Shadow-Config.md

```diff
@@ -27,16 +27,15 @@ hosts:
 - [`general.data_directory`](#generaldata_directory)
 - [`general.heartbeat_interval`](#generalheartbeat_interval)
 - [`general.log_level`](#generallog_level)
+- [`general.parallelism`](#general_parallelism)
```

Comment:
The other options don't have an underscore between the `general` keyword and the name of the option. I have no idea why not, it's harder to read without the underscore. I assume there was a reason? Do we want to be consistent?

Reply:
Good catch! Yeah, we don't control how the anchors are generated for the headings, and unfortunately it omits the '.', so this was a broken link.